tasks/windows_resources.py

```diff
@@ -10,36 +10,42 @@
 @task
 def build_messagetable(
     ctx,
+    target='pe-x86-64',
+    host_target='',
 ):
     """
     Build the header and resource for the MESSAGETABLE shared between agent binaries.
     """
-    windres_target = 'pe-x86-64'
-
     messagefile = MESSAGESTRINGS_MC_PATH
 
     root = os.path.dirname(messagefile)
 
     # Generate the message header and resource file
-    command = f"windmc --target {windres_target} -r {root} -h {root} {messagefile}"
+    windmc = "windmc"
+    if host_target:
+        windmc = host_target + "-" + windmc
+
+    command = f"{windmc} --target {target} -r {root} -h {root} {messagefile}"
     ctx.run(command)
 
-    build_rc(ctx, f'{root}/messagestrings.rc')
+    build_rc(ctx, f'{root}/messagestrings.rc', target=target, host_target=host_target)
 
 
-def build_rc(ctx, rc_file, vars=None, out=None):
+def build_rc(ctx, rc_file, vars=None, out=None, target='pe-x86-64', host_target=''):
     if vars is None:
         vars = {}
 
-    windres_target = 'pe-x86-64'
-
     if out is None:
         root = os.path.dirname(rc_file)
         out = f'{root}/rsrc.syso'
 
     # Build the binary resource
     # go automatically detects+includes .syso files
-    command = f"windres --target {windres_target} -i {rc_file} -O coff -o {out}"
+    windres = "windres"
+    if host_target:
+        windres = host_target + "-" + windres
```

Comment:
I thought that `host_target` was a weird name to use to change the executable, but I see that `--target` is a common mingw toolchain option. Can you add a docstring for the arg and provide the example/expected input of `x86_64-w64-mingw32` ?